frontend/src/components/common/Header.tsx

```diff
@@ -1,57 +1,166 @@
 import {
+  Button,
   Container,
   Drawer,
   DrawerBody,
   DrawerContent,
-  DrawerHeader,
   DrawerOverlay,
   Flex,
-  Icon,
   IconButton,
+  Image,
   Link,
   Stack,
-  Text,
   useDisclosure,
 } from "@chakra-ui/react";
-import React from "react";
-import { Link as ReactLink } from "react-router-dom";
+import React, { useContext, useEffect } from "react";
+import { Link as ReactLink, useHistory } from "react-router-dom";
 
-import { LOGIN_PAGE } from "../../constants/Routes";
-import { MenuIcon } from "./icons";
+import authAPIClient from "../../APIClients/AuthAPIClient";
+import {
+  HOME_PAGE,
+  LANDING_PAGE,
+  LOGIN_PAGE,
+  SCHEDULING_PAGE,
+} from "../../constants/Routes";
+import AuthContext from "../../contexts/AuthContext";
+import { CloseIcon, MenuIcon } from "./icons";
 
 const Header = (): JSX.Element => {
+  const history = useHistory();
   const { isOpen, onOpen, onClose } = useDisclosure();
+  const { authenticatedUser, setAuthenticatedUser } = useContext(AuthContext);
+
+  const onLogOutClick = async () => {
+    const success = await authAPIClient.logout(authenticatedUser?.id);
+    if (success) {
+      setAuthenticatedUser(null);
+    }
+  };
 
   return (
-    <Container pt="0.5rem">
-      <Flex>
+    <Container pt="1.5rem" maxWidth={{ base: "default", md: "70%" }}>
+      <Flex
+        pt={{ base: "0.5rem", md: "2rem" }}
+        flexDirection="row"
+        justifyContent="space-between"
+        alignItems="center"
+        display={{ base: "inline", md: "flex" }}
+      >
         <IconButton
           w="24px"
           h="24px"
           flex="1"
           aria-label="menu options"
           onClick={onOpen}
           backgroundColor="transparent"
+          display={{ base: "inline", md: "none" }}
         >
           <MenuIcon />
         </IconButton>
-        <Text pl="21px" flex="9">
-          Community Fridge
-        </Text>
+        <Image
+          objectFit="none"
+          src="header-logo.png"
+          alt="Community Fridge logo"
+          display="inline"
+        />
+        <Stack
+          spacing="2rem"
+          direction="row"
+          display={{ base: "none", md: "flex" }}
+        >
+          <Link as={ReactLink} to={LANDING_PAGE}>
+            Home
```

Comment:
```suggestion
          <Link as={ReactLink} to={LANDING_PAGE} onClick={onClose}>
            Home
```

for each of these links, can we add in a close so that the drawer closes after they are pressed?